WordPress/Jetpack/Info.plist

```diff
@@ -8,6 +8,10 @@
 	<string>${WP_APP_KEYCHAIN_ACCESS_GROUP}</string>
 	<key>WPPushNotificationAppID</key>
 	<string>${WP_PUSH_NOTIFICATION_APP_ID}</string>
+	<key>WPEventNamePrefix</key>
+	<string>jpios</string>
+	<key>WPExplatPlatform</key>
+	<string>wpios</string>
```

Comment:
Matches what was defined in WordPress/Jetpack/AppConstants.swift

```swift
@objc static let eventNamePrefix = "jpios"
@objc static let explatPlatform = "wpios"
```

Reply:
It's curios that it was defined like that though... 🤔 

But apparently, it was intentional: https://github.com/wordpress-mobile/WordPress-iOS/pull/20607